garage/contrib/exp/agents/agent.py

```diff
@@ -0,0 +1,43 @@
+import abc
+
+
+class Agent(abc.ABC):
+    @abc.abstractmethod
+    def get_actions(self, obs):
```

Comment:
i think it would be permissible to include the singular `get_action()` (gets one action only), as a convenience.

i think you can implement it in this ABC since the implementation is obvious